Intersect.Client.Framework/Gwen/Control/Base.cs

```diff
@@ -4330,7 +4330,7 @@ public virtual void DragAndDrop_Hover(Package p, int x, int y)
     // receiver
     public virtual bool DragAndDrop_CanAcceptPackage(Package p)
     {
-        return false;
+        return true;
```

Comment:
This should be false. It being true means a random label or button would be returning "true" for `CanAcceptPackage` which is totally wrong